contrib/completion/zsh/_docker

```diff
@@ -2122,7 +2122,7 @@ __docker_subcommand() {
                 "($help)--ipv6[Enable IPv6 networking]" \
                 "($help -l --log-level)"{-l=,--log-level=}"[Logging level]:level:(debug info warn error fatal)" \
                 "($help)*--label=[Key=value labels]:label: " \
-                "($help)--live-restore[Enable live restore of docker when containers are still running]" \
+                "($help)--live-restore[Enable live restore of docker when containers are still running. This option is not supported on Windows.]" \
```

Comment:
Do you mean the completions should be aware of the daemon context?
I was already planning to selectively enable experimental features in bash completion depending on the output of a `docker version` call. We could as well use the response in order to hide linux specific features if the daemon runs on Windows. But I'm not sure if I will be able to express the logic in a way that does not turn the script into a complet mess. Shell is quite limited here.